vignettes/messydates.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "Working with messy dates"
+author: "Henrique Sposito"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Working with messy dates}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Why `{messydates}`?
+
+Dates are often messy.
+Whether historical (or ancient), future, or even recent,
+we often only know approximately when an event occurred,
+that it happened within a particular period,
+an unreliable source means a date should be flagged as uncertain,
+or sources offer multiple, competing dates.
+
+The goal of `{messydates}` is to help with this problem by retaining and working
+with various kinds of date imprecision.
+\pkg{messydates} contains a set of tools for constructing and coercing into and from the 'mdate' class.
+This date class implements ISO 8601-2:2019(E) and allows regular dates to be annotated to express unspecified date components, approximate or uncertain date components, date ranges, and sets of dates.
+
+# Working with messydates: The size of the popuklation of the Roman Empire
```

Comment:
Spelling. Also, why is the roman_population dataset preferred over the battles dataset? What was wrong with `battles`? What does `roman_population` add?